Makefile

```diff
@@ -39,6 +49,36 @@ fmt-check:
 # For convenience. If this passes then it should pass in GitHub
 ci: fmt-check quiet-test
 
+#--------------------------------------------------------------------
+
+clean-data:
+	@rm -rf $(DATA_DIR)
+
+# Avoid a "feels like a bad day.." violation
+dummy-config:
+	@mkdir -p $(DATA_DIR)/config/policy
+	@echo '{"non_blocking_checks":["not_useful"]}' | jq > $(DATA_DIR)/config/policy/data.json
+
+# Set IMAGE as required like this:
+#   make fetch-attestation IMAGE=<someimage>
+#
+# The format and file path is intended to match what is used in the
+# verify-attestation-with-policy script in the build-definitions repo
+# so you can test your rules as they would be applied by the
+# verify-enterprise-contract task.
+#
+ifndef IMAGE
+  IMAGE="quay.io/lucarval/tekton-test:latest"
```

Comment:
Should the public key be added to this repo?

Reply:
No, I don't care about checking the sig. Just doing a `cosign download`.

Reply:
Oh my bad. I thought `cosign download` would check signature too. This is great.